src/context/ServiceInfoContext.tsx

```diff
@@ -74,17 +113,28 @@ const ServiceInfoProvider = ({ children }: React.PropsWithChildren<{}>) => {
         }
       }
 
-      const fetch = Api.getSession({ signal })
+      const fetch = Api.getSession({ signal, token: currentWallet?.token })
         .then((res) => (res.ok ? res.json() : Api.Helper.throwError(res)))
         .then((data: JmSessionData) => {
           const {
             session: sessionActive,
             maker_running: makerRunning,
             coinjoin_in_process: coinjoinInProgress,
             wallet_name: walletNameOrNoneString,
+            offer_list: offers,
+            schedule,
+            nickname,
           } = data
           const activeWalletName = walletNameOrNoneString !== 'None' ? walletNameOrNoneString : null
-          return { sessionActive, makerRunning, coinjoinInProgress, walletName: activeWalletName } as ServiceInfo
+          return {
```

Comment:
Should we add `singleCollaborativeTransactionRunning` and `schedulerRunning` directly to `ServiceInfo`? I feel like this could be useful throughout the app (e.g., #269). Maybe even something similar to what we have in `WalletContext` where there's a summary computed from the available raw data? What do you think?

Reply:
Good idea!

Reply:
Hmm.. thinking about it.., `makerRunning` and `coinjoinInProgress` can be updated via Websocket data.. `schedule` will currently only update from data of an http response.

I guess this deserves some more attention and docs.. will do it in a follow-up PR!

Reply:
Ok sounds good!